src/poliastro/plotting/_base.py

```diff
@@ -116,6 +119,20 @@ def __plot_coordinates_and_position(self, trajectory):
 
         return trace_coordinates, trace_position
 
+    def __plot_coordinates_and_position_anim(self, trajectory):
```

Comment:
Write the private methods using a single under score `_` instead of two `__`.

Reply:
yes sure